webapp-client/src/component/CategoryList.js

```diff
@@ -1,30 +1,74 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import ErrorMessage from './ErrorMessage';
 
```

Comment:
I know theres the .proptypes function, would it help documentation wise to have a comment explaining the props this component is expecting? or nah?

Reply:
I actually had comments explaining them before I had proptypes, and then our hosts recommended replacing comments with proptypes haha